src/Zoxive.HttpLoadTesting.Client/Domain/HttpStatusResult/Repositories/TestResultRepository.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+using Dapper;
+using Zoxive.HttpLoadTesting.Client.Domain.HttpStatusResult.Dtos;
+using Zoxive.HttpLoadTesting.Client.Pages;
+
+namespace Zoxive.HttpLoadTesting.Client.Domain.HttpStatusResult.Repositories
+{
+    public class TestResultRepository : ITestResultRepository
+    {
+        private const string Sql = @"SELECT
+ HttpStatusResult.[IterationId] as Id
+,HttpStatusResult.IterationId
+,HttpStatusResult.IterationId as Method
+,SUM(HttpStatusResult.[ElapsedMilliseconds]) AS ElapsedMilliseconds
+,TestName as RequestUrl
+,200 as StatusCode
+,MIN(HttpStatusResult.[RequestStartedMs]) AS [RequestStartedMs]
+FROM HttpStatusResult
+INNER JOIN Iteration ON Iteration.Id = HttpStatusResult.IterationId
+";
+
+        private readonly IDbConnection _dbConnection;
+
+        public TestResultRepository(IDbReader dbConnection)
+        {
+            _dbConnection = dbConnection.Connection;
+        }
+
+        public async Task<IReadOnlyCollection<HttpStatusResultDto>> GetTests(Filters filters)
```

Comment:
Think it was a refactoring change because I was splitting out the request/test repos. It looks like it is no longer needed so I have reverted back to IEnumerable.